orientdb/src/main/java/site/ycsb/db/OrientDBClient.java

```diff
@@ -1,272 +1,364 @@
 /**
  * Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
```

Comment:
update to add 2020 to the copyrights

Reply:
done - and I meanwhile updated all to 2021, too. Kindly merge, before we get into 2022.